term_proposal/YSHsieh7777/README.md

```diff
@@ -28,6 +28,56 @@ People often fall into **panic** when they meet dangers like fire or earthquake.
  - Calculate the number of people that die in the fire. 
  - Store the **best** traffic flow.
 
+## Project Structure
+
+Written with `C++` because it needs a large amount of computations. If it's hard to visualize the simulation with `C++`, `C++`(for large amount of computations) + `Pybind11` + `Python`(for user input processing and visualization).
+
+The program will need a **long loop** to update map and people's conditions, so a very efficient way to update conditions is necessary. It means that I need a well data structure that is easy to traverse, compute and update. Also, parallel programming is a possible strategy to improve performance.
```

Comment:
One way to deal with it is a data object with pipeline.  If you want to go parallel the data object can provide some basic infrastructure but you should need some design (layered design) in it as well.  Let's see how it would work.